libs/perun/components/src/lib/back-button/back-button.component.ts

```diff
@@ -2,6 +2,7 @@ import {Component} from '@angular/core';
 import {Location} from '@angular/common';
 
 @Component({
+  // tslint:disable-next-line:component-selector
   selector: 'app-back-button',
```

Comment:
```suggestion
  selector: 'perun-web-apps-back-button',
```
The tslint warnings are not there to be always disabled :wink: